js_modules/dagit/packages/app/package.json

```diff
@@ -22,11 +22,12 @@
   "devDependencies": {
     "@dagster-io/eslint-config": "workspace:*",
     "@dagster-io/react-scripts": "5.0.3",
+    "@types/eslint": "^8",
     "@types/jest": "^27.4.0",
     "@types/node": "^16.11.20",
     "@types/react": "17.0.4",
     "@types/react-dom": "^17.0.11",
-    "eslint": "8.7.0",
+    "eslint": "^8.6.0",
```

Comment:
Some weird dependency stuff going on here. Changing this so that `dagit-app` properly runs eslint.